packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -66,62 +71,59 @@ const validateOperationTask = (task, props) => {
   })
 }
 
-// Note: `bypassVdiChainsCheck` must be enabled because the XAPI might be not
-// able to coalesce VDIs as fast as the tests run.
-//
-// See https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
 describe('backupNg', () => {
-  let defaultBackupNg
-
-  beforeAll(() => {
-    defaultBackupNg = {
-      name: 'default-backupNg',
-      mode: 'full',
-      vms: {
-        id: config.vms.default,
-      },
-      settings: {
-        '': {
-          reportWhen: 'never',
-        },
-      },
-    }
-  })
-
   describe('.createJob() :', () => {
     it('creates a new backup job without schedules', async () => {
-      const backupNg = await xo.createTempBackupNgJob(defaultBackupNg)
+      const name = getDefaultName()
```

Comment:
Why isn't it handled by `createTempBackupNgJob`?

This forces everyone to deal with it, which is exactly what you're trying to avoid with default values?

Reply:
Yes but hard to test the output of the method.

Reply:
If `createTempBackupNgJob` mutates its `params`, you should be able to use it for your tests.

Reply:
Good idea :+1: